dbms/src/Flash/tests/gtest_interpreter.cpp

```diff
@@ -222,15 +208,12 @@ Union: <for test>
 Union: <for test>
  Expression x 10: <final projection>
   Expression: <projection>
-   Expression: <before projection>
-    Expression: <final projection>
-     Expression: <projection>
-      Expression: <before projection>
+   Expression: <final projection>
```

Comment:
Maybe clarify why the “before projection expression” is deleted

Reply:
due to https://github.com/pingcap/tiflash/pull/5326/files#diff-c21f466589627c568489360b256d315d80d6bf114ad7706fffc561183952172cR49